src/import/sb2import.js

```diff
@@ -226,11 +226,15 @@ var loadSound = function (sound, runtime) {
         log.error('No storage module present; cannot load sound asset: ', sound.md5);
         return;
     }
+    if (!runtime.audioEngine) {
+        log('No audio engine present; cannot load sound asset: ', sound.md5);
```

Comment:
These changed to `log.error` as part of the review process on #498 so I'd recommend changing this one too